.yamato/upm-ci.yml

```diff
@@ -104,19 +124,46 @@ test_trigger:
 {% endfor %}
 {% endfor %}
 
-nightly_test_trigger:
+release_4_2_nightly_test_trigger:
```

Comment:
is it possible to do 
`{% for release in nightly_tested_releases %}` here and

 `     {% for editor in {{release}}_nightly_trigger_editors %}`
later (currently line 136)?

The answer used to be no but they were looking into fixing that. If we can do indirect references like that we could save a lot of repetition.